wsl-builder/prepare-build/build.go

```diff
@@ -224,19 +241,53 @@ func downloadFile(url, dest string) (err error) {
 		return fmt.Errorf("http request failed with code %d", resp.StatusCode)
 	}
 
-	return writeContentInto(resp.Body, dest)
+	var size int
+	if size, err = strconv.Atoi(resp.Header.Get("Content-Length")); err != nil {
+		log.Printf("Warning: unknown size for %s: %v", url, err)
+	}
+	return writeContentInto(resp.Body, uint64(size), dest)
+}
+
+type WriteCounter struct {
```

Comment:
Consider documenting public types and functions.

Reply:
It should stay rather private, let me change it